src/main/kotlin/no/nav/klage/oppgave/domain/klage/KlagebehandlingAggregatFunctions.kt

```diff
@@ -234,6 +234,27 @@ object KlagebehandlingAggregatFunctions {
         return KlagebehandlingEndretEvent(klagebehandling = this, endringslogginnslag = listOfNotNull(endringslogg))
     }
 
+    fun Klagebehandling.setHjemlerInVedtak(
+        vedtakId: UUID,
+        nyVerdi: Set<Hjemmel>,
+        saksbehandlerident: String
+    ): KlagebehandlingEndretEvent {
+        val vedtak = getVedtakFromKlagebehandling(this, vedtakId)
+        val gammelVerdi = vedtak.hjemler
+        val tidspunkt = LocalDateTime.now()
+        vedtak.hjemler = nyVerdi.toMutableSet()
+        vedtak.modified = tidspunkt
+        val endringslogg =
+            endringslogg(
+                saksbehandlerident,
+                Felt.HJEMLER_I_VEDTAK,
+                gammelVerdi.map { it.id }.joinToString { "," },
+                nyVerdi.map { it.id }.joinToString { "," },
```

Comment:
Default separator er ", ". Om dette skal være lesbart (kun) for mennesker så tror jeg det er bedre.

Reply:
Jeg har vel tenkt at det skal være lesbart for maskiner også, selv om juryen kanskje ikke helt har bestemt seg der ennå.. :)